Makefile

```diff
@@ -8,6 +8,8 @@ MOD_VER_FILE=$(CURDIR)/kernel-modules/kobuild-tmp/MODULE_VERSION.txt
 LOCAL_SSH_PORT ?= 2222
 DEV_SSH_SERVER_KEY ?= $(CURDIR)/.collector_dev_ssh_host_ed25519_key
 
+ARCH ?= amd64
```

Comment:
Please move this line to `Makefile-constants.mk`, otherwise running commands directly in the `collector/` subdirectory will break.